components/cards.css

```diff
@@ -0,0 +1,102 @@
+.card{
+    width: 25rem;
+    min-height: 20rem;
+    border:1px solid black;
+    border-radius: 2%;
+    position: relative;
+
+}
+
+.card-header{
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding-left: 1rem;
+    padding-top: 1rem;
```

Comment:
This can be done by using short-hand property as  _padding: 1rem 0rem 0rem 1rem;_